docs/about/release_notes.rst

```diff
@@ -31,6 +27,119 @@ Release Notes
 Highlighted changes
 ===================
 
+Version 9.0
+-----------
+
+New workflow: Evaluate ensemble
+###############################
+
+In this version of ERT we have introduced a new simulation mode called: Evaluate ensemble.
+The main reason for this is that in the past, the Ensemble experiment simulation mode has had
+two distinct use cases, sensitivity analysis, and after doing a manual update step. But use
```

Comment:
"But use cases often ran in an existing experiment and ensemble, but had very different expectations
of what would happen to any parameters already in that experiment.
For the sensitivity analysis
case we would always want any existing parameters to be overwritten, however after doing a manual
update step the parameters should not be resampled."

Is this perhaps a bit easier to follow?

Perhaps the following is 
For instance, running a sensitivity analysis from an existing ensemble required re-sampling and overwriting existing parameters. However, if an `Ensemble experiment` was run after performing a manual update, the parameters should not be resampled.